Hets.cabal

```diff
@@ -134,6 +134,7 @@ Executable hets
       , http-types >= 0.6 && <= 1.0
       , text >= 0.5 && < 1.3
       , case-insensitive
+      , esqueleto >= 2.5.3
```

Comment:
Is 2.4.1 sufficient as well?

Reply:
2.4.1 is incompatible with persistent 2.7 and (more importantly) GHC 8.

Reply:
Bah. Is there somewhere a note about it? A changelog or similar stuff doesn't seem to exist it.

Reply:
I simply tried to downgrade to esqueleto 2.4.1 and compile Hets. Resolving the dependencies failed.

Reply:
There is no other note about compatibility. Unfortunately, they don't maintain changelogs.

Reply:
Ah ok, thought you have other info channels.
Did also some digging, but it seems that 2.4.1 needs persistent-"old" which probably breaks the stuff you already made. Anyway, at least bionic will have 2.5.3 and thus I'll add this to xenial PPA, too.